ldap/servers/slapd/back-ldbm/db-mdb/mdb_import.c

```diff
@@ -769,29 +769,12 @@ dbmdb_import_all_done(ImportJob *job, int ret)
     return ret;
 }
 
-/* Check if attrlist is empty or if attrname is in attrlist */
-static int
-is_reindexed_attr(const char *attrname, char **attrlist)
-{
-    if (!attrlist) {
-        return 1;
-    }
-    while (*attrlist) {
-        char *attr = *attrlist++;
-        /* Note that attrname is 'vlv#vlvIdx' while attrlist contains vlvIdx */
-        if (strcasecmp(attrname+4, attr) == 0) {
-            return 1;
-        }
-    }
-    return 0;
-}
-
 /* vlv_getindices callback that truncate vlv index (in reindex case) */
 static int
 truncate_index_dbi(struct attrinfo *ai, ImportCtx_t *ctx)
```

Comment:
The function truncate_index_dbi looks specific to vlv. Would it make sense to rename it with 'vlv' in its name ?